src/test/components/MarkerTooltipContents.test.js

```diff
@@ -53,205 +53,198 @@ describe('MarkerTooltipContents', function() {
 
     // Now add some markers to the profile.
     // Enumerate through all of the switch arms of the tooltip for coverage.
-    addMarkersToProfileReplacingSamples(
+    addMarkersToThreadReplacingSamples(profile.threads[0], [
       [
-        [
```

Comment:
This is diff churn from prettier changing indentation because of the re-ordering the arguments to have the thread first. I made this change to make it clear which thread I was operating on with the operation.